VultisigApp/VultisigApp/View Models/PhoneCheckUpdateViewModel.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  PhoneCheckUpdateViewModel.swift
+//  VultisigApp
+//
+//  Created by Amol Kumar on 2024-09-19.
+//
+
+import SwiftUI
+
+class PhoneCheckUpdateViewModel: ObservableObject {
+    @Published var showError: Bool = false
+    @Published var showDetails: Bool = false
+    @Published var showUpdateAlert: Bool = false
+    @Published var isUpdateAvailable: Bool = false
+    
+    @Published var latestVersionString: String = ""
+    @Published var currentVersionString: String = ""
+    
+    func resetData() {
+        showError = false
+        showDetails = false
+    }
+    
+    func checkForUpdates(isAutoCheck: Bool = false) {
+        let currentVersion = currentAppVersion()
+        let bundleID = Bundle.main.bundleIdentifier ?? ""
+        
+        fetchLatestAppStoreVersion(bundleID: bundleID) { latestVersion in
```

Comment:
should that be in viewModel?

Reply:
Isn't you are looking at a viewmodel?  😅